src/code/z_shrink_window.c

```diff
@@ -4,75 +4,76 @@
  */
 #include "prevent_bss_reordering.h"
 #include "global.h"
+#include "z64shrink_window.h"
 
 typedef struct {
     /* 0x0 */ s8 letterboxTarget;
     /* 0x1 */ s8 letterboxSize;
     /* 0x2 */ s8 pillarboxTarget;
     /* 0x3 */ s8 pillarboxSize;
-} ShrinkWindowContext; // size = 0x4
+} ShrinkWindow; // size = 0x4
 
-ShrinkWindowContext gShrinkWindowContext;
-ShrinkWindowContext* gShrinkWindowContextPtr;
+ShrinkWindow gShrinkWindow;
+ShrinkWindow* gShrinkWindowPtr;
```

Comment:
These are only used in this file, right? Can convert to `s`

Reply:
Ah, I was thinking `s` as in plural (end), not `s` as in local (start). Sounds good